CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Releases
 ========
 
+
+v1.20.1 (2017-10-20)
```

Comment:
Usually we'd do this when we do the release, I'd just put this stuff under `v1.21.0-dev` as it is on the dev branch for now, keep the version in `version.go` as `v1.21.0-dev`, just make the actual change, and when we do the release https://github.com/yarpc/yarpc-go/blob/dev/RELEASE.md we adjust the values in `CHANGELOG.md` and `version.go`.

Reply:
yep, but since this is a patch, we probably want to cut it asap, so bumped the version in the same change.